demo/src/components/UserInfoForm/UserInfoForm.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import {Form, Col} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+
+/*
+ * A template command to generate a basic text input form group.
+ * @param {string} label The label to display above the form.
+ * @param {!Form.Control} control The control element of the form.
+ * @param {boolean} [wholeCol=true] If this element is displayed on a
+ *      column by itself.
+ * @returns {!JSX} A form group with the given label and control.
+ */
+const makeFormGroup = (label, control, wholeCol=true) => {
```

Comment:
I think you mean "whole row", right? That or "single column".

I'd also be more explicit about the naming. Maybe "useWholeRow"?

Reply:
That's a better name, I'll change it